src/index.tsx

```diff
@@ -7,8 +7,8 @@
  */
 import './styles.css'
 import type { Component, ValidComponent } from 'solid-js'
```

Comment:
Can we remove this unused `ValidComponent` here?

Reply:
Resolved! Thanks.